manifests/tasks/check-model-and-containerfile-exists.yaml

```diff
@@ -27,18 +27,10 @@ spec:
 
       set -Eeuo pipefail
 
-      # Check model files exist
-      if [ -n "$(params.modelRelativePath)" ]; then
```

Comment:
In that case the `modelRelativePath` can be removed entirely from this task.

Reply:
👍